deps/build.jl

```diff
@@ -0,0 +1,28 @@
+using BinDeps
+using Compat
+
+@BinDeps.setup
+
+libmysqlclient = library_dependency("libmysqlclient",
+                                    alias=["libmysqlclient.so",
```

Comment:
>what do you mean by "anything with which I can start the mysql server" ? do you just need command line executables from the mysql package in addition to the library?

On Mac and Windows, you need to start the `mysql` server by executing a command-line executable. Otherwise, `mysql_connect` would fail, since there's no process running. 

> there is an unpacked_dir option for Binaries that might help

I tried this, and it didn't work. It still extracted into filename `mysql-5.7.15-win$arch`.